bot-twitter/src/authorizations.ts

```diff
@@ -0,0 +1,15 @@
+import getSigner from 'auctions-core/src/signer';
+import { authorizeWallet, getWalletAuthorizationStatus } from 'auctions-core/src/authorizations';
+
+export async function setupWallet(network: string) {
+    const signer = await getSigner(network);
+    const walletAddress = await signer.getAddress();
+    const isAuth = await getWalletAuthorizationStatus(network, walletAddress);
+    if (isAuth) {
+        console.info(`Wallet "${walletAddress}" has already been authorized`);
+        return;
+    }
+    console.info(`Wallet "${walletAddress}" has not been authorized yet. Attempting authorization now`);
+    const transactionHash = await authorizeWallet(network, false);
+    console.info(`Wallet "${walletAddress}" successfully authorized via "${transactionHash}" transaction`);
```

Comment:
What happens if the authorization fails?

Reply:
`authorizeWallet` will throw an error that will be propagated up to the `setup`, where it will throw unhandled error and halt bot execution. This is fine for this PR, although in the #7 I propose to do authorisation only when it's profitable to do so, therefore it will be part of the loop in the future